### PR TITLE
Resolve #1480, #1485, #1491, #1481, #1482, #1464, #1492, #1493

### LINKED ISSUE #1480: [Test] Fix emailTransferForm tests
```
 FAIL  src/tests/e2e/emailTransferForm.test.e2e.js (148.449s)
  Email transfer form tests
    ✓ Fill everything correctly (Ethereum) (7880ms)
    ✕ Fill everything correctly (DAI) (35100ms)
    ✕ Fill everything correctly (Bitcoin) (35306ms)
    ✕ Ethereum account should have two coins (5485ms)
    ✕ Bitcoin account should have one coin (5490ms)
    ✕ Transfer amount error tests (34465ms)
    ✓ send message === security answer error tests (8435ms)
    ✕ short security answer error  (7628ms)

  ● Email transfer form tests › Fill everything correctly (DAI)

    TimeoutError: Navigation timeout of 30000 ms exceeded

      81 | 
      82 |   beforeEach(async () => {
    > 83 |     await Promise.all([
         |     ^
      84 |       page.waitForNavigation({
      85 |         waitUntil: 'networkidle0'
      86 |       }),

      at ../../../node_modules/puppeteer/lib/LifecycleWatcher.js:142:21
      at FrameManager.waitForFrameNavigation (../../../node_modules/puppeteer/lib/FrameManager.js:139:19)
      at Frame.waitForNavigation (../../../node_modules/puppeteer/lib/FrameManager.js:414:12)
      at Page.waitForNavigation (../../../node_modules/puppeteer/lib/Page.js:692:12)
          at async Promise.all (index 0)
      at Object.<anonymous> (emailTransferForm.test.e2e.js:83:5)
        -- ASYNC --
      at Frame.<anonymous> (../../../node_modules/puppeteer/lib/helper.js:111:15)
      at Page.waitForNavigation (../../../node_modules/puppeteer/lib/Page.js:692:49)
      at Page.<anonymous> (../../../node_modules/puppeteer/lib/helper.js:112:23)
      at Object.waitForNavigation (emailTransferForm.test.e2e.js:84:12)
          at runMicrotasks (<anonymous>)
        -- ASYNC --
      at Page.<anonymous> (../../../node_modules/puppeteer/lib/helper.js:111:15)
      at Object.waitForNavigation (emailTransferForm.test.e2e.js:84:12)
          at runMicrotasks (<anonymous>)

  ● Email transfer form tests › Fill everything correctly (Bitcoin)

    TimeoutError: Navigation timeout of 30000 ms exceeded

      81 | 
      82 |   beforeEach(async () => {
    > 83 |     await Promise.all([
         |     ^
      84 |       page.waitForNavigation({
      85 |         waitUntil: 'networkidle0'
      86 |       }),

      at ../../../node_modules/puppeteer/lib/LifecycleWatcher.js:142:21
      at FrameManager.waitForFrameNavigation (../../../node_modules/puppeteer/lib/FrameManager.js:139:19)
      at Frame.waitForNavigation (../../../node_modules/puppeteer/lib/FrameManager.js:414:12)
      at Page.waitForNavigation (../../../node_modules/puppeteer/lib/Page.js:692:12)
          at async Promise.all (index 0)
      at Object.<anonymous> (emailTransferForm.test.e2e.js:83:5)
        -- ASYNC --
      at Frame.<anonymous> (../../../node_modules/puppeteer/lib/helper.js:111:15)
      at Page.waitForNavigation (../../../node_modules/puppeteer/lib/Page.js:692:49)
      at Page.<anonymous> (../../../node_modules/puppeteer/lib/helper.js:112:23)
      at Object.waitForNavigation (emailTransferForm.test.e2e.js:84:12)
          at runMicrotasks (<anonymous>)
        -- ASYNC --
      at Page.<anonymous> (../../../node_modules/puppeteer/lib/helper.js:111:15)
      at Object.waitForNavigation (emailTransferForm.test.e2e.js:84:12)
          at runMicrotasks (<anonymous>)

  ● Email transfer form tests › Ethereum account should have two coins

    expect(received).toEqual(expected)

    Expected value to equal:
      2
    Received:
      5

      225 |       await emtPage.updateForm('account', { walletType: walletType, platformType: platformType })
      226 |       const coinList = await emtPage.getCoinList()
    > 227 |       expect(coinList.length).toEqual(2)
          |                               ^
      228 |     },
      229 |     timeout
      230 |   )

      at Object.toEqual (emailTransferForm.test.e2e.js:227:31)
          at runMicrotasks (<anonymous>)

  ● Email transfer form tests › Bitcoin account should have one coin

    Evaluation failed: TypeError: Cannot read property 'textContent' of null

      at ../../../__puppeteer_evaluation_script__:5:71
          at Array.map (<anonymous>)
      at ../../../__puppeteer_evaluation_script__:2:20
      at ExecutionContext._evaluateInternal (../../../node_modules/puppeteer/lib/ExecutionContext.js:122:13)
          at runMicrotasks (<anonymous>)
      at ExecutionContext.evaluate (../../../node_modules/puppeteer/lib/ExecutionContext.js:48:12)
      at JSHandle.evaluate (../../../node_modules/puppeteer/lib/JSHandle.js:54:12)
      at ElementHandle.$$eval (../../../node_modules/puppeteer/lib/JSHandle.js:522:20)
      at DOMWorld.$$eval (../../../node_modules/puppeteer/lib/DOMWorld.js:167:19)
      at EmailTransferFormPage.getCoinList (pages/emailTransferForm.page.js:16:20)
      at Object.<anonymous> (emailTransferForm.test.e2e.js:240:24)
        -- ASYNC --
      at ExecutionContext.<anonymous> (../../../node_modules/puppeteer/lib/helper.js:111:15)
      at JSHandle.evaluate (../../../node_modules/puppeteer/lib/JSHandle.js:54:42)
      at JSHandle.<anonymous> (../../../node_modules/puppeteer/lib/helper.js:112:23)
      at ElementHandle.$$eval (../../../node_modules/puppeteer/lib/JSHandle.js:522:38)
          at runMicrotasks (<anonymous>)
      at DOMWorld.$$eval (../../../node_modules/puppeteer/lib/DOMWorld.js:167:19)
      at EmailTransferFormPage.getCoinList (pages/emailTransferForm.page.js:16:20)
      at Object.<anonymous> (emailTransferForm.test.e2e.js:240:24)
        -- ASYNC --
      at ElementHandle.<anonymous> (../../../node_modules/puppeteer/lib/helper.js:111:15)
      at DOMWorld.$$eval (../../../node_modules/puppeteer/lib/DOMWorld.js:167:34)
          at runMicrotasks (<anonymous>)
      at EmailTransferFormPage.getCoinList (pages/emailTransferForm.page.js:16:20)
      at Object.<anonymous> (emailTransferForm.test.e2e.js:240:24)
        -- ASYNC --
      at Frame.<anonymous> (../../../node_modules/puppeteer/lib/helper.js:111:15)
      at Page.$$eval (../../../node_modules/puppeteer/lib/Page.js:355:29)
      at Page.<anonymous> (../../../node_modules/puppeteer/lib/helper.js:112:23)
      at EmailTransferFormPage.$$eval [as getCoinList] (pages/emailTransferForm.page.js:16:31)
      at Object.<anonymous> (emailTransferForm.test.e2e.js:240:24)

  ● Email transfer form tests › Transfer amount error tests

    TimeoutError: Navigation timeout of 30000 ms exceeded

      81 | 
      82 |   beforeEach(async () => {
    > 83 |     await Promise.all([
         |     ^
      84 |       page.waitForNavigation({
      85 |         waitUntil: 'networkidle0'
      86 |       }),

      at ../../../node_modules/puppeteer/lib/LifecycleWatcher.js:142:21
      at FrameManager.waitForFrameNavigation (../../../node_modules/puppeteer/lib/FrameManager.js:139:19)
      at Frame.waitForNavigation (../../../node_modules/puppeteer/lib/FrameManager.js:414:12)
      at Page.waitForNavigation (../../../node_modules/puppeteer/lib/Page.js:692:12)
          at async Promise.all (index 0)
      at Object.<anonymous> (emailTransferForm.test.e2e.js:83:5)
        -- ASYNC --
      at Frame.<anonymous> (../../../node_modules/puppeteer/lib/helper.js:111:15)
      at Page.waitForNavigation (../../../node_modules/puppeteer/lib/Page.js:692:49)
      at Page.<anonymous> (../../../node_modules/puppeteer/lib/helper.js:112:23)
      at Object.waitForNavigation (emailTransferForm.test.e2e.js:84:12)
          at runMicrotasks (<anonymous>)
        -- ASYNC --
      at Page.<anonymous> (../../../node_modules/puppeteer/lib/helper.js:111:15)
      at Object.waitForNavigation (emailTransferForm.test.e2e.js:84:12)
          at runMicrotasks (<anonymous>)

  ● Email transfer form tests › Transfer amount error tests

    expect(received).toBe(expected) // Object.is equality

    Expected: true
    Received: false

      279 |       await emtPage.updateForm('currencyAmount', { currencyAmount: '.' })
      280 |       expect(await emtPage.formInputFieldToBe('cryptoAmount', { existOnly: true })).toBe(false)
    > 281 |       expect((await emtPage.getTextFieldStatus('cryptoAmount')).error).toBe(true)
          |                                                                        ^
      282 |       expect((await emtPage.getTextFieldStatus('cryptoAmount')).helperText).toMatch(
      283 |         /Please enter a valid amount/
      284 |       )

      at Object.toBe (emailTransferForm.test.e2e.js:281:72)

  ● Email transfer form tests › short security answer error 

    expect(received).toMatch(expected)

    Expected value to match:
      /We recommend you to use auto-generated security answer/
    Received:
      "What's Security Answer?"

      339 | 
      340 |       expect((await emtPage.getTextFieldStatus('securityAnswer')).error).toBe(false)
    > 341 |       expect((await emtPage.getTextFieldStatus('securityAnswer')).helperText).toMatch(
          |                                                                               ^
      342 |         /We recommend you to use auto-generated security answer/
      343 |       )
      344 |     },

      at Object.toMatch (emailTransferForm.test.e2e.js:341:79)
          at runMicrotasks (<anonymous>)

Test Suites: 1 failed, 1 total
Tests:       6 failed, 2 passed, 8 total
Snapshots:   0 total
Time:        148.5s
```

### PR DESCRIPTION
**Pull request by timothywangdev**

### CODE REVIEW

File: src/configureStore.js [hunk 1]
```diff
@@ -17,14 +17,15 @@ const history = createBrowserHistory()
...
   const middlewares = [
     routerMiddleware(history),
-    trackerMiddleware,
     errorMiddleware,
     promiseMiddleware(),
     thunk
   ]
 
-  if (process.env.NODE_ENV === `development`) {
+  if (process.env.REACT_APP_ENV === `test`) {
```
Comment (InVincible2016) [inline src/configureStore.js:25]: use typedEnv if possible

File: config-overrides.js [hunk 2]
```diff
@@ -8,6 +8,12 @@ module.exports = {
     // ...add your webpack config
     // suppress grpc warnings
     config.module.exprContextCritical = false
+
+    // remove hot-loader
+    if (process.env.REACT_APP_HOT_LOADING === 'false') {
```
Comment (InVincible2016) [inline config-overrides.js:13]: use typedEnv if possible

File: src/components/TransferFormComponents/TransferAmountTextField.jsx [hunk 3]
```diff
@@ -61,18 +61,18 @@ export default function TransferAmountTextField (props: Props) {
...
     const decimals = getCryptoDecimals(cryptoType)
 
-    // if the value has been set to '', either
+    // if the value has been set to '' or null, either
     // caused by account changes, or by deleting all chars
     // in the text field by the user
     //
     // treat it as correct value in this case
-    if (value === '') return
+    if (!value || value === '') return
```
Comment (InVincible2016) [inline src/components/TransferFormComponents/TransferAmountTextField.jsx:75]: !value should be enough. '' is false

File: config-overrides.js (hunk 2 above)
Comment (timothywangdev) [reply config-overrides.js:13]: cannot use typed env for webpack config file

**Pull request merged**